Source/embeddings/index.ts

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+export {
+    CannotRegisterEmbeddingThatIsNotAClass,
+    compare,
+    CompareDecoratedMethod,
+    CompareDecoratedMethods,
+    DeleteDecoratedMethod,
+    DeleteDecoratedMethods,
+    deleteMethod,
+    embedding,
+    EmbeddingAlreadyHasACompareDecorator,
+    EmbeddingAlreadyHasACompareMethod,
+    EmbeddingAlreadyHasADeleteDecorator,
+    EmbeddingAlreadyHasADeleteMethod,
+    EmbeddingBuilder,
+    EmbeddingBuilderForReadModel,
+    EmbeddingClassBuilder,
+    EmbeddingClassCompareMethod,
+    EmbeddingClassDeleteMethod,
+    EmbeddingClassOnMethod,
+    EmbeddingDecoratedType,
+    EmbeddingDecoratedTypes,
+    EmbeddingsBuilder,
+    EmbeddingsBuilderCallback,
+    ICanBuildAndRegisterAnEmbedding,
+    on,
+    OnDecoratedEmbeddingMethod,
+    OnDecoratedEmbeddingMethods,
+    ReadModelAlreadyDefinedForEmbedding
+} from './Builder';
+export { Embedding } from './Embedding';
+export { EmbeddingCompareCallback } from './EmbeddingCompareCallback';
+export { EmbeddingContext } from './EmbeddingContext';
+export { EmbeddingDeleteCallback } from './EmbeddingDeleteCallback';
+export { EmbeddingId } from './EmbeddingId';
+export { EmbeddingProjectCallback } from './EmbeddingProjectCallback';
+export { EmbeddingProjectContext } from './EmbeddingProjectContext';
+export { Embeddings } from './Embeddings';
+export { IEmbedding } from './IEmbedding';
+export { IEmbeddings } from './IEmbeddings';
+export * as internal from './Internal';
```

Comment:
Can't we just export things from here as we do above with { }?

Reply:
The names could potentially clash right? Like this:
```typescript
export { EventProcessor } from './EventProcessor';
export { EventProcessor } from './Internal';
```

Reply:
I see. We could keep this for now.